django_s3_storage/storage.py

```diff
@@ -312,15 +312,17 @@ def url(self, name):
         # All done!
         return url
 
-    def accessed_time(self, name):
-        return self.modified_time(name)
-
-    def created_time(self, name):
-        return self.modified_time(name)
-
     def modified_time(self, name):
         return make_naive(self.meta(name)["LastModified"], utc)
 
+    created_time = accessed_time = modified_time
+
+    def get_modified_time(self, name):
+        timestamp = self.meta(name)["LastModified"]
+        return timestamp if settings.USE_TZ else make_naive(timestamp, utc)
```

Comment:
So I guess the `utc` needs taking out of here.

Aside from that, I'm happy to merge this.